dev/fattest.simplicity/src/componenttest/containers/ArtifactoryImageNameSubstitutor.java

```diff
@@ -30,9 +30,9 @@ public class ArtifactoryImageNameSubstitutor extends ImageNameSubstitutor {
     private static final Class<?> c = ArtifactoryImageNameSubstitutor.class;
 
     /**
-     * Manual override that will allow builds or users to pull from Artifactory instead of DockerHub.
+     * Manual override that will allow builds or users to pull from the default registry instead of Artifactory.
      */
-    private static final String substitutionOverride = "fat.test.use.artifactory.substitution";
+    private static final String substitutionOverride = "fat.test.artifactory.force.external.repo";
```

Comment:
I would suggest renaming this variable to `forceExternal` or `forceNoArtifactory` or `forceNoSubstitution`